Dev6/Dev6/CommandFactory.cs

```diff
@@ -0,0 +1,93 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Dev6
+{
+    /// <summary>
+    /// Class for creates commands
+    /// </summary>
+    class CommandFactory
+    {
+        public List<CarCollection> Catalogs { get; private set; }
+        private Action ExecuteCommands { get; set; }
+        private ICommand Command { get; set; }
+
+        /// <summary>
+        /// Constructor initializes fields
+        /// </summary>
+        /// <param name="carCollection">Catalog of cars</param>
+        public CommandFactory(List<CarCollection> carCollection)
+        {
+            this.Catalogs = carCollection;
+        }
+
+        /// <summary>
+        /// Gets command
+        /// </summary>
+        public void GetCommand()
```

Comment:
please use single responsibility. The method 'GetCommand' should only return desired command. But in this case it reads info from command line, choose a command and then execute them